shared/src/commonMain/kotlin/io.projectnewm.shared/login/LoginFeature.kt

```diff
@@ -0,0 +1,12 @@
+package io.projectnewm.shared.login
+
+import io.ktor.client.*
+import io.projectnewm.shared.KtorClientFactory
+
+object LoginFeature {
```

Comment:
While we figure our DI tooling, we can use this class to test out the endpoints.